src/packages/verticaltabs/verticaltabs.tsx

```
@@ -0,0 +1,228 @@
+    }
+  }
+  useEffect(() => {
+    let index = titles.current.findIndex((t) => t.value === value)
+    index = index < 0 ? 0 : index
+    setTimeout(() => {
+      scrollIntoView(index)
+    })
+  }, [value])
+
+  const tabChange = (item: TabsTitle) => {
+    onClick && onClick(item.value)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)